ctk/common/src/test/kotlin/org/codice/compilance/verification/core/response/CoreLogoutResponseProtocolVerifierSpec.kt

```diff
@@ -57,37 +58,37 @@ class CoreLogoutResponseProtocolVerifierSpec : StringSpec() {
 
     init {
         "logout response with correct second-level status code should pass" {
-            Common.buildDom(createLogoutResponse(SUCCESS)).let {
+            NodeDecorator(Common.buildDom(createLogoutResponse(SUCCESS))).let {
                 CoreLogoutResponseProtocolVerifier(logoutRequest, it, HTTP_POST, SUCCESS).verify()
             }
         }
 
         "logout response with incorrect second-level status code should fail" {
-            Common.buildDom(createLogoutResponse(RESPONDER)).let {
+            NodeDecorator(Common.buildDom(createLogoutResponse(RESPONDER))).let {
                 shouldThrow<SAMLComplianceException> {
                     CoreLogoutResponseProtocolVerifier(logoutRequest, it, HTTP_POST, SUCCESS)
                             .verify()
-                }.apply {
-                    this.message?.shouldContain(SAMLCore_3_7_3_2_b.message)
-                    this.message?.shouldContain(SAMLCore_3_7_3_2_d.message)
+                }.message?.apply {
+                    this.shouldContain(SAMLCore_3_7_3_2_b.message)
+                    this.shouldContain(SAMLCore_3_7_3_2_d.message)
                 }
             }
         }
 
         "logout response with no second-level status code when expected should fail" {
-            Common.buildDom(createLogoutResponse(null)).let {
+            NodeDecorator(Common.buildDom(createLogoutResponse(null))).let {
                 shouldThrow<SAMLComplianceException> {
                     CoreLogoutResponseProtocolVerifier(logoutRequest, it, HTTP_POST, SUCCESS)
                             .verify()
-                }.apply {
-                    this.message?.shouldContain(SAMLCore_3_7_3_2_b.message)
-                    this.message?.shouldContain(SAMLCore_3_7_3_2_d.message)
+                }.message?.apply {
```

Comment:
✏️ You can take out the `this.` before `this.shouldContain()` in the `apply` block.